### DISCUSSION TITLE
Direct Edit client app cannot be opened

### DISCUSSION THREAD

**Discussion by NiklasBr** [pimcore/pimcore#17569] [Q&A / Support]
Clients see this error when trying to open the Mac OS app downloaded [from the documentations site](https://pimcore.com/docs/platform/Direct_Edit/Pimcore_Direct_Edit_Client).
![Screenshot 2024-09-05 at 11 56 27](https://github.com/user-attachments/assets/2dae1bd9-34a2-4117-bc24-cdfa8b4d3cd5)

There is an undocumented work-around to open unsigned apps in the current version of Mac OS which involves right-clicking (ctrl-clicking) the app binary to open the context menu and then choosing "Open…" but that option will [go away in the next version of Mac OS](https://arstechnica.com/gadgets/2024/08/macos-15-sequoia-makes-you-jump-through-more-hoops-to-disable-gatekeeper-app-checks/) (to become even more complicated) which will be released later this year.

What makes it even more important to fix is that many MDM solutions actively block unsigned apps, making it impossible to use any of the above two workarounds.

Best solution would be to sign the binary to avoid these problems. Are there any plans on doing so?

**Comment by fashxp** [comment]
Yes, we are working on that one to the binaries signed.
